Library/Homebrew/commands.rb

```diff
@@ -108,10 +108,15 @@ def internal_developer_commands_paths
   end
 
   def official_external_commands_paths(quiet:)
-    %w[bundle services test-bot].map do |cmd|
-      tap = Tap.fetch("Homebrew/#{cmd}")
+    OFFICIAL_CMD_TAPS.flat_map do |tap_name, cmds|
+      tap = Tap.fetch(tap_name)
       tap.install(quiet: quiet) unless tap.installed?
-      external_ruby_v2_cmd_path(cmd)
+      cmds.map do |cmd|
+        # TODO: ideally these would all be Ruby V2 commands (for improved
+        # command parsing and because `brew-command-not-found-init.rb` outputs
+        # when `require`d)
+        external_ruby_v2_cmd_path(cmd) || external_ruby_cmd_path(cmd) || external_cmd_path(cmd)
```

Comment:
Just curious, how much of an "ideally" is this? Worth spending time on?

It looks like the only ones that aren't Ruby V2 are `autoupdate` and `command-not-found-init`, right?

Reply:
@Rylan12 I think definitely worth spending time on 👍🏻. Yeh, those are the only ones I noticed, too.

Reply:
Looking at the syntax used in bundle/etc I don't think it'd be too difficult for me to rewrite autoupdate in the new style. I was semi-aware of it already but didn't have much incentive to do a rewrite as an external tap.